api/reqs/tests/model_tests.py

```diff
@@ -28,3 +31,12 @@ def test_original_url():
 
     policy.document_source = Mock(url='http://example.com/uploaded')
     assert policy.original_url == 'http://example.com/uploaded'
+
+
+@pytest.mark.django_db
+def test_slug_is_created_from_title_on_save_if_slug_is_empty():
+    policy = mommy.make(models.Policy)
+    policy.title = 'hello there'
+    policy.slug = ''
+    policy.save()
```

Comment:
A few small details that might make this slightly easier to read in the future:
* `mommy.make` will _save_ the record to the database, which might not be intended. `mommy.prepare` will not
* `mommy.make` and `.prepare` allows us to set model properties via kwargs

I think that could leads us to something like:
```python
policy = mommy.prepare(models.Policy, title='hello there', slug='')
policy.save()  # not using `make` to emphasize the save() method
assert policy.slug == 'hello-there'
```
What do you think?

Reply:
Oh great idea! I'm unfamiliar w/ model mommy so I didn't know about the `make` vs. `prepare` distinction, thanks for explaining it.